npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "perseo",
-  "version": "0.4.1-next",
+  "version": "0.5.0-next",
```

Comment:
Having adjusted this "by hand" probably mean that the release.sh stepping script should take also npm-shrinkwrap.json into account and currently it is not doing it.

@dmoranj could you have a look this? Maybe it also affect to fiware-orion-pep...